packages/ag-charts-enterprise/src/features/crosshair/crosshair.ts

```diff
@@ -158,10 +160,10 @@ export class Crosshair extends _ModuleSupport.BaseModuleInstance implements _Mod
             line.lineDash = lineDash;
             line.lineDashOffset = lineDashOffset;
 
-            line.y1 = 0;
-            line.y2 = isVertical ? bounds.height : 0;
-            line.x1 = 0;
-            line.x2 = isVertical ? 0 : bounds.width;
+            line.y1 ||= 0;
+            line.y2 ||= isVertical ? bounds.height : 0;
+            line.x1 ||= 0;
+            line.x2 ||= isVertical ? 0 : bounds.width;
```

Comment:
Should this be `??` instead of `||`, or is the intent to always replace `0` values too?

Reply:
we want to replace falsy values including 0 as they are initialized with 0